### DISCUSSION TITLE
sandbox internet issue

### DISCUSSION THREAD

**Discussion by HavocShreds** [gucci-on-fleek/lockdown-browser#67] [Q&A]
I have a problem where the sandbox does not connect to the internet using hotspot or even usb tethering. If anyone knows a workaround for this please let me know.

**Comment by gucci-on-fleek** [comment]
You might have to do some weird bridging stuff. See https://github.com/gucci-on-fleek/lockdown-browser/issues/49#issuecomment-1974681947 and #58, although I'm not sure if any of those will work here.

But I'd check #65 and https://github.com/gucci-on-fleek/lockdown-browser/discussions/53#discussioncomment-8909229 before trying this since many people have been having issues lately.
